public/index.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
```

Comment:
Why do you add these comment?

Reply:
It was recommended when running: composer cs public/index.php

Reply:
It is misuse. We have three config files for php-cs-fixer.
Please use `composer cs-fix` and `composer cs` without argument.

The comment is not needed. Please remove.